docs/tutorials/getting_started.rst

```diff
@@ -0,0 +1,290 @@
+===============
+Getting Started
+===============
+
+Installation
+============
+
+Qiskit Experiments is built on top of Qiskit, so we recommend that you first install
+Qiskit following its :external+qiskit:doc:`installation guide <getting_started>`. 
+Qiskit Experiments supports the same platforms and Python versions (currently 3.7+) as 
+Qiskit itself.
+
+Qiskit Experiments releases can be installed via the python package manager 
```

Comment:
```suggestion
Qiskit Experiments releases can be installed via the Python package manager: 
```

Reply:
Fixed the capitalization but the colon isn't right since ``pip`` is on the next line.